### PR TITLE
add core-management to core release

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -71,6 +71,11 @@
       <artifactId>azure-core-http-okhttp</artifactId>
       <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;current} -->
     </dependency>
+    <dependency>
```
Comment (JimSuplizio) [inline eng/jacoco-test-coverage/pom.xml:74]: Thank you @ChenTanyi!

**Comment by ChenTanyi** [comment]
@JimSuplizio Is this PR OK?

**Comment by ChenTanyi** [comment]
@JonathanGiles Is there any concern for the core-management joining to azure-core pipeline?
@srnagar @alzimmermsft Is there any problem for helping releasing azure-core-management when you release the azure-core?

If all are OK, I'll merge it. Thanks!

**Comment by ChenTanyi** [comment]
Thanks @alzimmermsft @samvaity. The release time could depend on the Azure Core libs, currently the `azure-core-management` has no big change pending. cc @weidongxu-microsoft @xccc-msft @yungezz

**Pull request merged**